src/fidesops/schemas/saas/strategy_configuration.py

```diff
@@ -0,0 +1,28 @@
+from typing import Union
+
+from pydantic import BaseModel
+
+
+class StrategyConfiguration(BaseModel):
+    """Base class for strategy configuration"""
+
+    pass
```

Comment:
I would like to keep this `StrategyConfiguration` class so that we can accurately describe this data class, when we don't yet know which specific strategy is used:

```
class PostProcessorStrategyData(BaseModel):
    """Data class for postprocessor strategy"""

    strategy: str
    configuration: StrategyConfiguration
```

Reply:
thanks! I see now that you've added this to the `PostProcessorStrategyData.configuration`